cmd/coap/main.go

```diff
@@ -43,6 +43,8 @@ func main() {
 
 	go func() {
 		coapAddr := fmt.Sprintf(":%d", cfg.Port)
+		startMsg := fmt.Sprintf("CoAP adapter service started, exposed port %d", cfg.Port)
```

Comment:
No need for adding startup message to CoAP adapter. @dusanb94 will ensure it is present as part of #168.